ArduCopter/mode_auto.cpp

```diff
@@ -148,6 +148,15 @@ bool ModeAuto::allows_arming(AP_Arming::Method method) const
     return (copter.g2.auto_options & (uint32_t)Options::AllowArming) != 0;
 };
 
+bool ModeAuto::allows_weathervaning(void) const
+{
```

Comment:
If you want to this function can be removed.  It's just a hang over from changes I made last time I updated the PR to master.